src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,37 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    public static final int CAPACITY = 12;
+    public static final int START_VALUE = -1;
+    public static final int MAX = 11;
+    private final int[] numbers = new int[CAPACITY];
+    public static final int RETURN = -1;
+
+    public int total = START_VALUE;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).
This constant should be different (separate) than the one in peekaboo and countOut methods.

Reply:
Task 10 in "accessory i hermetyzacja" concerned 'total' getter.